crates/sui-json-rpc/src/move_utils.rs

```diff
@@ -237,8 +249,9 @@ impl MoveUtilsServer for MoveUtils {
                         _ => MoveFunctionArgType::Pure,
                     })
                     .collect::<Vec<MoveFunctionArgType>>()),
-                None => Err(Error::SuiRpcInputError(SuiRpcInputError::GenericNotFound(
-                    format!("No parameters found for function {}", function),
+                None => Err(ClientError::NotFoundCustom(format!(
```

Comment:
nit: There's an `InvalidParam` error, so maybe this could also be a `NoFunctionParam` error?